mlir/lib/Analysis/DataFlow/IntegerRangeAnalysis.cpp

```diff
@@ -36,10 +36,26 @@
 using namespace mlir;
 using namespace mlir::dataflow;
 
+namespace {
+
+OptionalIntRanges getOptionalRange(const IntegerValueRange &range) {
```

Comment:
Nit: I think this'd usually be a `static` and not `namespace`d

Reply:
Removed these functions